### PR TITLE
Add postgres instrumentation

### PR DESCRIPTION
**Pull request by voldern**
Resolves https://github.com/openzipkin/zipkin-js/issues/149

### CODE REVIEW

File: packages/zipkin-instrumentation-postgres/test/integrationTest.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    const postgres = new (getPostgres(tracer).Client)(postgresConnectionOptions);
+    postgres.connect();
+
+    postgres.query('INVALID QUERY', (firstError) => {
+      postgres.query('ERROR QUERY').catch((secondError) => {
+        const query = new Postgres.Query('FAILED QUERY()');
+        const result = postgres.query(query);
+        result.on('error', (thirdError) => {
+          const annotations = recorder.record.args.map(args => args[0]);
+          expectErrorAnnotation(annotations[5], firstError);
```
Comment (adriancole) [inline packages/zipkin-instrumentation-postgres/test/integrationTest.js]: maybe describe what this is doing here? ex are these errors in different spans? (ps 'error' is often a binary annotation aka tag)

File: packages/zipkin-instrumentation-postgres/test/integrationTest.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+          runTest(annotations.slice(10, 15));
+
+          expect(
+            annotations[0].traceId.spanId
+          ).not.to.equal(annotations[5].traceId.spanId);
+          expect(
+            annotations[0].traceId.spanId
+          ).not.to.equal(annotations[10].traceId.spanId);
+
+          annotations.forEach(ann => {
```
Comment (adriancole) [inline packages/zipkin-instrumentation-postgres/test/integrationTest.js]: style, but I'd find this stuff easier to understand if in span format vs a list of annotations. For example, you can look at batch-recorder.test.js

Comment (voldern) [reply packages/zipkin-instrumentation-postgres/test/integrationTest.js]: Based it on https://github.com/openzipkin/zipkin-js/blob/master/packages/zipkin-instrumentation-memcached/test/integrationTest.js.

But this approach seems much clearer and succinct. Updated tests now.

File: packages/zipkin-instrumentation-postgres/test/integrationTest.js (hunk 1 above)
Comment (voldern) [reply packages/zipkin-instrumentation-postgres/test/integrationTest.js]: Good catch! Fixed it.

**Comment by adriancole** [comment]
@brianc fyi in zipkin-land we usually merge things after there's a few folks interested. This is a client-side trace integration (using your library). What this means is that postgres time and metadata  will end up included in traces that originate from anywhere (even an upstream browser request). Just in case you are interested.

**Pull request merged**